docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -0,0 +1,264 @@
+
+### Observe that the Node begins using the new configuration
+
+Retrieve the Node with `kubectl get node NODE_NAME -o yaml`, and look for the `ConfigOK` condition in `status.conditions`. You should the message `Using current (UID: CONFIG_MAP_UID)` when the Kubelet starts using the new configuration.
+
+For convenience, you can use the following command (using `jq`) to filter down to the `ConfigOK` condition:
+
+```
+kubectl get no NODE_NAME -o json | jq '.status.conditions|map(select(.type=="ConfigOK"))'
+```
+
+If something goes wrong, you may see one of several different error conditions, detailed in the Table of ConfigOK Conditions, below. When this happens, you should check the Kubelet's log for more details.
+
+### Edit the configuration file again
+
+To change the configuration again, we simply repeat the above workflow. Try editing the `kubelet` file, changing the previously changed parameter to a new value.
+
+### Push the newly edited configuration to the control plane
+
+Push the new configuration to the control plane in a new ConfigMap with the following command:
+
+`kubectl create configmap my-Node-config --namespace=kube-system --from-file kubelet --append-hash -o yaml`
+
+This new ConfigMap will get a new name, as we have changed the contents. We will refer to the new name as NEW_CONFIG_MAP_NAME and the new uid as NEW_CONFIG_MAP_UID.
+
+### Once more configure the Node to use the new configuration
+
+Once more, edit the Node's `spec.configSource` with `kubectl edit Node NODE_NAME`. Your new `spec.configSource` should look like the following, with names substituted as necessary:
+
+```
+configSource:
+    configMapRef:
+        name: NEW_CONFIG_MAP_NAME
+        namespace: kube-system
+        uid: NEW_CONFIG_MAP_UID
+```
+
+### Once more observe that the Kubelet is using the new configuration
+
+Once more, retrieve the Node with `kubectl get node NODE_NAME -o yaml`, and look for the `ConfigOK` condition in `status.conditions`. You should the message `Using current (UID: NEW_CONFIG_MAP_UID)` when the Kubelet starts using the new configuration.
+
+### Reset the Node to use the default configuration again
+
+Finally, if you wish to reset the Node to use the configuration it was provisioned with, simply edit the Node with `kubectl edit Node NODE_NAME` and remove the `spec.configSource` subfield.
+
+After removing this subfield, you should eventually observe that the ConfigOK condition's message reverts to either `using current (default)` or `using current (init)`, depending on how the Node was provisioned.
+{% endcapture %}
+
+{% capture discussion %}
+## Understanding ConfigOK Conditions
+
+The following table describes several of the `ConfigOK` Node conditions you might encounter in a cluster that has Dynamic Kubelet Config enabled. If you observe a condition with `status=False`, you should check the Kubelet log for more error details by searching for the message or reason text.
+
+<table>
```

Comment:
Why not use markdown tables? Would be much easier to review

Reply:
Newlines are significant in markdown tables, so any cell that contains a lot of text would have to be on the same line.